robot/impl/local_robot_test.go

```diff
@@ -653,6 +653,10 @@ func (da *dummyArm) JointPositions(ctx context.Context, extra map[string]interfa
 	return nil, errors.New("fake error")
 }
 
+func (da *dummyArm) ModelFrame() referenceframe.Model {
```

Comment:
These tests wouldn't break if we used the inject.Arm struct for this instead of doing something separate.  We could merge this as is but its just asking to get broken again down the line